test/python/tf_unittest_runner.py

```diff
@@ -185,5 +197,14 @@ def run_test(test_list, verbosity=2):
         test_result = unittest.TextTestRunner(verbosity=verbosity).run(tests)
 
 
+def read_tests_from_file(filename, verbosity=2):
+    with open('tests_to_run.txt') as list_of_tests:
```

Comment:
Should here be `with open(filename)`?

Reply:
Also, `verbosity` is not required as an input for this function

Reply:
Thanks for pointing out @mingshan-wang. Made this change and removed verbosity.